File: C++/Beecrowd/bee-2057.cpp
#include <iostream>
using namespace std;

int main(){
    int S, T, F;
    cin >> S >> T >> F;
    int ans = S + T + F;
    if(ans >= 24){ ans -= 24; }
    if(ans < 0){ ans += 24; }
    cout << ans << endl;

    return 0;
}

/* 2057 - Time Zone
https://www.beecrowd.com.br/judge/en/problems/view/2057 */

File: Python/Beecrowd/bee-2057.py
S, T, F = map(int, input().split())
ans = S + T + F
if ans >= 24:
    ans -= 24
if ans < 0:
    ans += 24
print(ans)

# 2057 - Time Zone
# https://www.beecrowd.com.br/judge/en/problems/view/2057